browser/resources/settings/brave_settings_overrides.js

```diff
@@ -1,9 +1,331 @@
+    const autofillEl = getMenuElement(templateContent, '/autofill')
+    const privacyEl = getMenuElement(templateContent, '/privacy')
+    privacyEl.insertAdjacentElement('afterend', autofillEl)
+    // Remove extensions link
+    const extensionsLinkEl = templateContent.querySelector('#extensionsLink')
+    if (!extensionsLinkEl) {
+      console.error('[Brave Settings Overrides] Could not find extensionsLinkEl to remove')
+    }
+    extensionsLinkEl.remove()
+    // Add version number to 'about' link
+    const aboutEl = templateContent.querySelector('#about-menu')
+    if (!aboutEl) {
+      console.error('[Brave Settings Overrides] Could not find about-menun element')
+    }
+    const aboutTitleContent = aboutEl.innerHTML
+    aboutEl.innerHTML = `
+      <div class="brave-about-graphic">
+        <iron-icon icon="brave_settings:full-color-brave-lion"><iron-icon>
+      </div>
+      <div class="brave-about-meta">
+        <span class="brave-about-item brave-about-menu-link-text">${aboutTitleContent}</span>
+        <span class="brave-about-item brave-about-menu-version">v ${loadTimeData.getString('braveProductVersion')}</span>
+      </div>
+    `
+  },
+  'settings-basic-page': (templateContent) => {
+    // Routes
+    const r = settings.router.routes_
+    if (!r.BASIC) {
+      console.error('[Brave Settings Overrides] Routes: could not find BASIC page')
+    }
+    r.GET_STARTED = r.BASIC.createSection('/getStarted', 'getStarted')
+    r.SHIELDS = r.BASIC.createSection('/shields', 'shields')
+    r.SOCIAL_BLOCKING = r.BASIC.createSection('/socialBlocking', 'socialBlocking')
+    r.EXTENSIONS = r.BASIC.createSection('/extensions', 'extensions')
+    r.BRAVE_SYNC = r.BASIC.createSection('/braveSync', 'braveSync')
+    if (!r.SITE_SETTINGS) {
+      console.error('[Brave Settings Overrides] Routes: could not find SITE_SETTINGS page')
+    }
+    r.SITE_SETTINGS_AUTOPLAY = r.SITE_SETTINGS.createChild('autoplay')
+    // Autofill route is moved to advanced,
+    // otherwise its sections won't show up when opened.
+    if (!r.AUTOFILL || !r.ADVANCED) {
+      console.error('[Brave Settings Overrides] Could not move autofill route to advanced route', r)
+    } else {
+      r.AUTOFILL.parent = r.ADVANCED
+    }
+    // Add 'Getting Started' section
+    // Entire content is wrapped in another conditional template
+    const actualTemplate = templateContent.querySelector('template')
+    if (!actualTemplate) {
+      console.error('[Brave Settings Overrides] Could not find basic-page template')
+      return
+    }
+    const basicPageEl = actualTemplate.content.querySelector('#basicPage')
+    if (!basicPageEl) {
+      console.error('[Brave Settings Overrides] Could not find basicPage element to insert Getting Started section')
+    } else {
+      const sectionsFromTop = document.createElement('div')
+      sectionsFromTop.innerHTML = `
+        <template is="dom-if" if="[[showPage_(pageVisibility.getStarted)]]">
```

Comment:
can we load this from an html file instead of inlining?

Reply:
In this case, using a separate module adds a bit more complexity since all the logic for determining which page's are visible at a given time (given current url, search filtering, etc) are all in the `basic_page` module. Those properties and methods could probably be passed as attributes to another module. We can at least split this up in to a separate file (e.g `brave_settings_sections`)